docs/source/checks/nlp/train_test_validation/plot_embeddings_drift.py

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8 -*-
+"""
+.. _nlp__embeddings_drift:
+
+Embeddings Drift
+*******************
+
+This notebooks provides an overview for using and understanding the embeddings
+drift check.
+
+**Structure:**
+
+* `What Is Embeddings Drift? <#what-is-a-embeddings-drift>`__
+* `Loading the Data <#loading-the-data>`__
+* `Run the Check <#run-the-check>`__
+
+What Is Embeddings Drift?
+==============================
+
+Drift is simply a change in the distribution of data over time, and it is
+also one of the top reasons why machine learning model's performance degrades
+over time.
+
+In unstructured data such as text, we cannot measure the drift of the data directly, as there's no "distribution"
+to measure. In order to measure the drift of the data, we can use the model's embeddings as a proxy for the data
+distribution.
+
+For more on embeddings, see the `NLP Embeddings Guide
+<https://docs.deepchecks.com/stable/nlp/usage_guides/nlp_embeddings.html>`_.
+
+How Deepchecks Detects Dataset Drift
+------------------------------------
+
+This check detects the embeddings drift by using :ref:`a domain classifier <drift_detection_by_domain_classifier>`.
+Other methods to detect drift include :ref:`univariate measures <drift_detection_by_univariate_measure>`
+which is used in other checks, such as :ref:`Feature Drift check <tabular__feature_drift>`.
+"""
+
+#%%
+from deepchecks.nlp.datasets.classification import tweet_emotion
+from deepchecks.nlp.checks import TextEmbeddingsDrift
+
+#%%
+# Load Data
+# ==========
+# For this example, we'll use the tweet emotion dataset, which is a dataset of tweets labeled by one of four emotions:
```

Comment:
Add the line for calculating / setting embeddings here, and link to the "how to calculate embeddings" guide you wrote.

Reply:
Would that work in a comment? I don't think so...

Reply:
would work if done  as part of the rst block before the code.